modules/odc-configure-web-terminal-image-session.adoc

```diff
@@ -0,0 +1,22 @@
+
+// Module is included in the following assemblies:
+//
+// * web_console/web_terminal/configuring-web-terminal.adoc
+
+:_content-type: PROCEDURE
+[id="odc-configure-web-terminal-image-session_{context}"]
+= Configuring the web terminal image for a session
+
+You can change the default image for the web terminal for your current session.
+
+.Prerequisites
+
+* You have access to an {product-title} cluster that has the {web-terminal-op} installed.
+* You are logged into the web console.
+
+.Procedure
+
+. Click the web terminal icon (image:odc-wto-icon.png[title="wto icon"]).
+. Click the *Image* link to display advanced configuration options for the web terminal image.
```

Comment:
```suggestion
. Click *Image* to display advanced configuration options for the web terminal image.
```
Per ISG: "In general, refer to the link by its link text. If you must distinguish the link from other interface elements to avoid confusion, refer to the link by its link text followed by link." https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#link